tests/providers/salesforce/hooks/test_salesforce.py

```diff
@@ -86,23 +106,56 @@ def test_get_conn_password_auth(self, mock_salesforce):
         mock_salesforce.assert_called_once_with(
             username=password_auth_conn.login,
             password=password_auth_conn.password,
-            security_token=extras["extra__salesforce__security_token"],
-            domain=extras["extra__salesforce__domain"],
+            security_token=extras.get('extra__salesforce__security_token') or extras.get('security_token'),
```

Comment:
this looks like it might need to be updated after the last update removing the short param names

Reply:
Correct, I missed this